IPython/core/interactiveshell.py

```diff
@@ -924,7 +924,9 @@ def init_virtualenv(self):
         
         # In Cygwin paths like "c:\..." and '\cygdrive\c\...' are possible
         if str(p_venv).startswith("\\cygdrive"):
-            p_venv = "C:" / Path(str(p_venv)[11:])
+            p_venv = Path(str(p_venv))
```

Comment:
Do we still need to `Path`→`str`→`Path` here?

Reply:
Good catch! No, we don't need it. I'll remove it.